content/posts/how-redefining-software-quality-can-save-your-day.mdx

```diff
@@ -92,7 +90,7 @@ Hey, don't forget that quality processes in software are vastly inspired from ol
 
 [Introduced by Bill Smith at Motorola](https://en.wikipedia.org/wiki/Six_Sigma), Six Sigma qualifies processes that lead to a 99.99966% free of defects production.
 
-<img class="floating-right" src="/assets/blog/how-redefining-software-quality-can-save-your-day/om_droit_au_but.png" alt="Olympique de Marseille soccer team baseline" width="100"/>
+<img className="floating-right" src="/assets/blog/how-redefining-software-quality-can-save-your-day/om_droit_au_but.png" alt="Olympique de Marseille soccer team baseline" width="100"/>
```

Comment:
is it because of MDX & img React component? :thinking:

Reply:
Yes